changes/probe.py

```diff
@@ -58,11 +68,20 @@ def has_changelog():
 
 
 def has_readme():
-    """README.md"""
+    """README"""
+    readme = None
+    message = 'Create a README file'
+    for filename in os.listdir("."):
+        readme = re.match(r"README($|\.\w{2,})", filename, re.I)
```

Comment:
I fixed up everything else, so tell me your thoughts on this and I'll push the fixes.